content/en/blog/2024/otel-operator-q-and-a/index.md

```diff
@@ -0,0 +1,345 @@
+however, they can be installed standalone, which means that you don’t need to
+install the Prometheus Operator just to use these two CRs with the Target
+Allocator.
+
+The easiest way to install the
+[`PodMonitor`](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#monitoring.coreos.com/v1.PodMonitor)
+and
+[`ServiceMonitor`](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#monitoring.coreos.com/v1.ServiceMonitor)
+CRs is to grab a copy of the individual
+[PodMonitor YAML](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/charts/crds/crds/crd-podmonitors.yaml)
+and
+[ServiceMonitor YAML](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/charts/crds/crds/crd-servicemonitors.yaml)
+[custom resource definitions (CRDs)](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/),
+like this:
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.71.2/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml
+
+kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.71.2/example/prometheus-operator-crd/monitoring.coreos.com_podmonitors.yaml
+```
+
+Check out my example of the OpenTelemetry Operator’s Target Allocator with
+`ServiceMonitor`
+[here](https://github.com/avillela/otel-target-allocator-talk/tree/main?tab=readme-ov-file#3b--kubernetes-deployment-servicenow-cloud-observability-backend).
+
+### Q7: Do I need to create a service account to use the Target Allocator?
+
+No, but you do need to do a bit of extra work. So, here’s the deal…although you
+need a
+[service account](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/)
+to use the Target Allocator, you don’t have to create your own.
+
+If you enable the Target Allocator and don’t create a service account, one is
+automagically created for you. This service account’s default name is a
+concatenation of the Collector name (`metadata.name` in the
+`OpenTelemetryCollector` CR) and `-collector`. For example, if your Collector is
+called `mycollector`, then your service account would be called
+`mycollector-collector`.
+
+By default, this service account has no defined policy. This means that you’ll
+still need to create your own
+[`ClusterRole`](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-and-clusterrole)
+and
+[`ClusterRoleBinding`](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#rolebinding-and-clusterrolebinding),
+and associate the `ClusterRole` to the `ServiceAccount` via
+`ClusterRoleBinding`.
+
+See the
+[Target Allocator readme](https://github.com/open-telemetry/opentelemetry-operator/tree/main/cmd/otel-allocator#rbac)
+for more on Target Allocator RBAC configuration.
```

Comment:
I wonder if it's worth noting that we're planning to automate this fully in the near future. We already have it for some components, like the k8sprocessor, and there's an outstanding PR to do it automatically if the user gives the operator the necessary RBAC permissions to manage ClusterRoles and ClusterRoleBindings.

Reply:
Do you have a link to the PR @swiatekm-sumo?

Reply:
@swiatekm-sumo I've updated the text to reflect this. Do you have a link to the PR you mentioned above?

Reply:
Thanks! updated the post with the PR link!